### DISCUSSION TITLE
How can I include the latest three blog posts in the home page?

### DISCUSSION THREAD

**Discussion by dunithd** [thuliteio/doks#205] [Q&A] [answered]
Hello,

I need to include the latest three blog posts on the home page, just like https://getdoks.org/.

Appreciate it if someone can help me with the required configurations. 

P.S Refer to the attached screenshot taken from the homepage of https://getdoks.org/.

![Screen Shot 2021-04-03 at 12 11 42 AM](https://user-images.githubusercontent.com/178694/113444452-8e702080-9411-11eb-9028-ed47f6c81992.png)

**Comment by h-enk** [comment, marked as answer]
@dunithd, 

Copy lines 124..143 of https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html and paste them into your `./layouts/index.html`

**Reply by dunithd** [reply]
Thanks, it worked.

**Comment by pranavdhawale** [comment]
How can we show cover image of the blog in latest blog section???

**Reply by h-enk** [reply]
Copy lines 25..46 from `node_modules/@hyas/doks-core/layouts/blog/list.html` to `layouts/index.html` (replace lines 48..54) and there you go:

![2024-05-22_20-55-26](https://github.com/gethyas/doks/assets/3902872/fe307b27-74ca-4763-b9d9-075f5377a707)

**Reply by pranavdhawale** [reply]
Thanks man!

**Reply by h-enk** [reply]
You're welcome!
